aws-geo-location/src/main/java/com/amplifyframework/geo/location/AWSLocationGeoPlugin.kt

```diff
@@ -64,18 +64,26 @@ class AWSLocationGeoPlugin(
         configuration.searchIndices!!.default
     }
 
+    val credentialsProvider: AWSCredentialsProvider by lazy {
```

Comment:
this should probably be `private`

Reply:
the idea is that the `AWSLocationGeoPlugin` exposes its credentials provider so the map adapter can just use, instead of getting its own reference to the `Auth` plugin, something that the Geo plugin already does.